pkg/integration/components/git.go

```diff
@@ -14,6 +14,10 @@ func (self *Git) CurrentBranchName(expectedName string) *Git {
 	return self.assert("git rev-parse --abbrev-ref HEAD", expectedName)
 }
 
+func (self *Git) TagNamesAt(ref string, expectedNames []string) *Git {
+	return self.assert(fmt.Sprintf(`git tag --contains "%s"`, ref), strings.Join(expectedNames, "\n"))
```

Comment:
```suggestion
	return self.assert(fmt.Sprintf(`git tag --points-at "%s"`, ref), strings.Join(expectedNames, "\n"))
```
This is what we really want here